src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -72,6 +72,7 @@ export type FeatureFlags = {
   ChainflipDca: boolean
   PublicTradeRoute: boolean
   ThorFreeFees: boolean
+  Jupiter: boolean
```

Comment:
```suggestion
  JupiterSwap: boolean
```

Reply:
Done in [e4891d6](https://github.com/shapeshift/web/pull/8120/commits/e4891d6256da07fae14a9f7eaa0e89ba7aefbe0d)